xExtension-ReadingTime/static/readingtime.js

```diff
@@ -5,12 +5,14 @@
 		flux_list: null,
 		flux: null,
 		textContent: null,
-		words_count: null,
+		count: null,
 		read_time: null,
 		reading_time: null,
 
 		init: function () {
 			const flux_list = document.querySelectorAll('[id^="flux_"]');
+			const speed = window.context.extensions.reading_time_speed;
+			const metrics = window.context.extensions.reading_time_metrics;
```

Comment:
I found an issue that sometimes ReadingTime doesn't work with the following browser error.
It may be a race condition.

```console
ext.php?f=xExtension-ReadingTime%2Fstatic%2Freadingtime.js&t=js&1742290508:14 Uncaught TypeError: Cannot read properties of undefined (reading 'extensions')
    at Object.init (ext.php?f=xExtension-ReadingTime%2Fstatic%2Freadingtime.js&t=js&1742290508:14:33)
    at add_load_more_listener (ext.php?f=xExtension-ReadingTime%2Fstatic%2Freadingtime.js&t=js&1742290508:80:16)
    at reading_time (ext.php?f=xExtension-ReadingTime%2Fstatic%2Freadingtime.js&t=js&1742290508:87:3)
    at ext.php?f=xExtension-ReadingTime%2Fstatic%2Freadingtime.js&t=js&1742290508:91:2
```

Reply:
74233da fixed this.